cmd/ocm-backplane/accessrequest/createAccessRequest.go

```diff
@@ -0,0 +1,253 @@
+				if err == nil {
+					reasons := elevate.GetElevateContextReasons(config)
+					if len(reasons) > 0 {
+						options.reason = reasons[0]
+					}
+				} else {
+					logger.Warnf("won't extract the elevation reason from the kube context failed to read: %v", err)
+				}
+			}
+		}
+
+		if options.reason != "" {
+			fmt.Printf("Reason for elevations read from the kube config: %s\n", options.reason)
+			if strings.ToLower(utils.AskQuestionFromPrompt("Do you want to use this as the reason/justification for the access request to create (Y/n)? ")) == "n" {
+				options.reason = ""
+			}
+		}
+
+		if options.reason == "" {
+			options.reason = utils.AskQuestionFromPrompt("Please enter a reason/justification for the access request to create: ")
+		}
+
+		if options.reason == "" {
+			return errors.New("no reason/justification; consider using the --reason option with a non empty string")
+		}
+	}
+
+	jiraProject := "SDAINT"
+	if strings.HasPrefix(ocmConnection.URL(), ProductionOCMUrl) {
+		jiraProject = "OHSS"
+	}
+
+	jiraURLPrefix := utils.JiraBaseURL + "/browse/"
+	notificationIssueID := options.notificationIssueID
+	var issueService *jira.IssueService
+	var notificationIssue *jira.Issue
+
+	if strings.HasPrefix(options.notificationIssueID, jiraURLPrefix) {
+		notificationIssueID = options.notificationIssueID[len(jiraURLPrefix):]
+	}
+
+	if len(notificationIssueID) > 0 {
+		if !strings.HasPrefix(notificationIssueID, jiraProject+"-") {
+			return fmt.Errorf("issue '%s' passed with the --notification-issue option does not belong to the %s JIRA project", options.notificationIssueID, jiraProject)
+		}
+
+		jiraClient, err := utils.GetJiraClient()
+
+		if err != nil || jiraClient == nil {
+			logger.Warnf("won't verify the validity of the '%s' JIRA issue as the client failed to be created: %v", options.notificationIssueID, err)
+		} else {
+			issueService = jiraClient.Issue
+
+			if issueService == nil {
+				logger.Warnf("won't verify the validity of the '%s' JIRA issue as there is no issue service in the client", options.notificationIssueID)
+			} else {
+				notificationIssue, _, err = issueService.Get(notificationIssueID, nil)
+				if err != nil || notificationIssue == nil {
+					return fmt.Errorf("the '%s' JIRA issue is invalid: %v", options.notificationIssueID, err)
+				}
+			}
+		}
+	} else {
+		jiraClient, err := utils.GetJiraClient()
+
+		if err != nil || jiraClient == nil {
+			return fmt.Errorf("failed create the JIRA client, consider creating the issue separately and passing it with the --notification-issue option: %v", err)
+		}
+
+		issueService = jiraClient.Issue
+
+		if issueService == nil {
+			return errors.New("no issue service in the JIRA client, consider creating the issue separately and passing it with the --notification-issue option")
+		}
+
+		issueType := "Story"
+		if jiraProject == "OHSS" {
+			issueType = "Incident"
+		}
+
+		notificationIssue = &jira.Issue{
+			Fields: &jira.IssueFields{
+				Description: "To be defined",
```

Comment:
Let's just put "Access request tracker" ?

Reply:
Could we also add some labels so we can do filter easily?

Reply:
@Nikokolas3270 Not good at naming, but how about `approve_access_request`?

Reply:
As said this description is (normally) be changed a few milliseconds later. So whatever we put won't matter much. Lets put something short & neutral as it will appear in the JIRA ticket history